projects/addon-tablebars/project.json

```
@@ -21,15 +21,15 @@
             ]
         },
         "test": {
-            "builder": "@nrwl/workspace:run-commands",
+            "builder": "nx:run-commands",
```

Should we also replace `builder` with `executor` ?

```suggestion
            "executor": "nx:run-commands",
```

---

fixed